docs/apache-airflow-providers-openlineage/guides/user.rst

```diff
@@ -263,6 +264,61 @@ a string of semicolon separated Airflow Operators full import paths to ``extract
 
   AIRFLOW__OPENLINEAGE__EXTRACTORS='full.path.to.ExtractorClass;full.path.to.AnotherExtractorClass'
 
+Enabling OpenLineage on DAG/task level
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One can selectively enable OpenLineage for specific DAGs and tasks by using the ``opt_in`` policy.
+To enable this policy, set the ``opt_in`` option to True in the [openlineage] section of your Airflow configuration file:
+
+.. code-block:: ini
+
+    [openlineage]
+    opt_in = True
```

Comment:
This needs to be updated.

Reply:
Yes, it does! Thanks for pointing this out.
Updated.